Engine/src/Engine/Rendering/Window.cpp

```diff
@@ -110,7 +115,8 @@ namespace Engine
 	
 	float Window::getAspectRatio() const
 	{
-		return m_camera.aspectRatio;
+		//return m_camera.aspectRatio;
+		return 9.f / 16.f;
```

Comment:
this can be in the .h

Reply:
can also be a constexpr

Reply:
Was using a fixed value to test, now reverted back to previous return value